### DISCUSSION TITLE
What is the minimum PHP version for this package?

### DISCUSSION THREAD

**Discussion by souravtah** [coderflexx/laravel-ticket#3] [Q&A] [answered]
What is the minimum PHP version for this package? Kindly mention it in the readme
My version of 8.0.3 PHP doesn't work.

**Comment by codebyray** [comment, marked as answer]
While it is not mentioned in the readme, if you click the ``conposer.json`` file you can see the minimums required.
```php
"require": {
    "php": "^8.1",
    "spatie/laravel-package-tools": "^1.13.0",
    "illuminate/contracts": "^9.0"
}
```

**Comment by ousid** [comment]
Hey @souravtah 

The requirements on this package is `^8.1`, I'll try to support `^8.x`, when I have time.

If you can make a PR for it, It will much appreciate it.

Edit:
I have tried to add PHP `^8.0` support #5, but this package uses built-in `enums` and it's only supports in PHP `^8.1` and upcoming versions
